syntax = "proto2";
import "session_event.proto";

message Event_ServerIdentification {
    extend SessionEvent {
        optional Event_ServerIdentification ext = 500;
    }
    optional string server_name = 1;
    optional string server_version = 2;
    optional uint32 protocol_version = 3;
}
